kernel/byterun/coq_interp.c

```diff
@@ -1322,11 +1336,13 @@ value coq_interprete
         CheckInt2();
         value divisor;
         divisor = *sp++;
-        if (uint63_eq0(divisor)) {
+        int b;
+        Uint63_eq0(b, divisor);
```

Comment:
Are we confident no allocation can kill `divisor` here? It might be better to adjust the stack later.